docs/guides/developer/docs/develop/development-tips.md

```diff
@@ -67,3 +69,129 @@ correctly be restored, after the new bundle has been deployed.
 
 
 
+Specific development environments tips
+--------------------------------------
+
+
+### Ubuntu
+
+#### Update System
+
+```sh
+$ apt update
+$ apt upgrade -y
+```
+
+#### Install Packages via APT
+
+```sh
+$ apt install -y git openjdk-11-jdk maven gcc g++ build-essential cmake curl sox hunspell synfig ffmpeg
+```
+
+#### Install NodeJS (optional)
+
+```sh
+$ curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+```
+
+#### Install and start Elasticsearch with Docker
+
+You can use `docker-compose` to easily run Elasticsearch:
+
+```sh
+$ cd docs/scripts/devel-dependency-containers
+$ docker-compose up -d
+```
+
+To shut the services down again, run:
+
+```sh
+$ cd docs/scripts/devel-dependency-containers
+$ docker-compose down
+```
+
+
+#### Set System Java JDK
+
+Choose the Java Version 11 by entering:
+
+```sh
+$ update-alternatives --config java
+```
+
+
+### Fedora
+
+
+#### Update System
+
+```sh
+$ dnf update -y
+```
+
+#### Install Dependencies
+
+```sh
+$ dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm -y
+$ dnf group install 'Development Tools'
```

Comment:
Why do you need this group?

Reply:
To be honest, is a copy from the original docs. I will delete it because I don't know if you still need it. Anyway, installing the RPM or following the Dev environment guide should be enough.